gradio/utils.py

```diff
@@ -44,6 +44,7 @@
 import anyio
 import gradio_client.utils as client_utils
 import httpx
+import matplotlib
```

Comment:
We shouldn't do this top-level import. Its a slow library to import and it slows down `import gradio as gr`